plugins/catalog-backend/src/modules/core/UrlReaderProcessor.ts

```diff
@@ -93,6 +93,8 @@ export class UrlReaderProcessor implements CatalogProcessor {
           value: parseResults as CatalogProcessorEntityResult[],
         });
       }
+
+      emit(processingResult.refresh(`${location.type}:${location.target}`));
```

Comment:
I'm thinking we want just the URL here rather than `<type>:<target>`, what do you think @backstage/maintainers?